### PR TITLE
Save to file

### LINKED ISSUE #69: Store daily medication list
Convert daily medication list into list of strings to be ready to save each row into today.txt

### PR DESCRIPTION
**Pull request by T0nyLin**
# Checklist for feature implementation and modification
Closes #69
- [x] Added relevant milestone (under milestone tab on the right)
- Implementation
  - [ ] Sources (if any) credited under both relevant section(s) and Developer Guide (Acknowledgement)
  - [ ] JUnit Test(s)
- Documentation
  - [x] JavaDoc for classes and methods
  - Developer Guide
    - [ ] Design (Can add as a todo statement for later on)
    - [ ] Implementation (Can add as a todo statement for later on)
    - [ ] Relevant User Story present
    - [ ] Non-functional requirement(s) that the feature addresses (if any)
    - [ ] Glossary of the (technical) term(s) used (if any)
    - [ ] Specialised instructions for manual testing (if any)
  - User Guide
    - [ ] Description and usage of feature (if any)
  - Contributions (under `/docs/team/*.md`)
    - [ ] Code/Feature/Enhancement contribution
    - [ ] Developer Guide contribution
    - [ ] User Guide contribution
    - [ ] Misc and team-based admin tasks (if any)

# Checklist for non-implementation related PR
Closes #69
- [ ] Added relevant milestone (under milestone tab on the right)
- Documentation (Contributions)
  - [ ] Developer Guide contributions (if any)
  - [ ] User Guide contributions (if any)
  - [ ] Review contributions (if any)
  - [ ] Miscellaneous and team-based admin tasks (if any)

# Additional Information

Nil

### CODE REVIEW

File: src/main/java/meditracker/DailyMedicationManager.java [hunk 1]
```diff
@@ -43,9 +40,14 @@ public DailyMedicationManager(MedicationManager medicationManager) {
      */
     public DailyMedicationManager(List<String> lines) {
         dailyMedications = new ArrayList<>();
-        for(String line : lines) {
-            DailyMedication dailyMedication = parseImportedLine(line);
-            addDailyMedication(dailyMedication);
+        ui = new Ui();
+        try {
+            for(String line : lines) {
```
Comment (nickczh) [inline src/main/java/meditracker/DailyMedicationManager.java:45]:
```suggestion
            for (String line : lines) {
```
Leave a space after for to enhance readability.

**Review by nickczh — CHANGES_REQUESTED** [review]
LGTM! Just a nit to work on.

File: src/main/java/meditracker/DailyMedicationManager.java (hunk 1 above)
Comment (T0nyLin) [reply src/main/java/meditracker/DailyMedicationManager.java]: Implemented in [b94f76f]

**Review by nickczh — APPROVED** [review]
LGTM!

**Pull request merged**